deploy-implicit-aurelia-app.cmd

```diff
@@ -0,0 +1,146 @@
+@if "%SCM_TRACE_LEVEL%" NEQ "4" @echo off
```

Comment:
TODO: Add deployment scripts for the other projects, so that we can have them live in Azure.